KoAR.Core/ItemMemoryInfo.cs

```diff
@@ -35,6 +35,9 @@ public ItemMemoryInfo(int typeIdOffset, int offset, int datalength, int coreEffe
                 ItemName = Encoding.Default.GetString(ItemBytes, Offsets.CustomNameText, NameLength);
             }
         }
+
+        public GameSave GameSave { get; }
```

Comment:
probably best as private readonly field. No one should be accessing this property on the item.